1-Computer-Vision/Exercises/1-3-1-Logistic-Regression/2022-08-27-Logistic-Regression.ipynb

```diff
@@ -80,7 +80,7 @@
   },
```

Comment:
<p>I see that this is quite a tremendous performance hit; hope you are able to regain access to your GPU server soon.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jonathanloganmoran/ND0013-Self-Driving-Car-Engineer/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>